NorthstarDLL/plugins/plugins.cpp

```diff
@@ -190,6 +190,12 @@ std::optional<Plugin> PluginManager::LoadPlugin(fs::path path, PluginInitFuncs*
 
 bool PluginManager::LoadPlugins()
 {
+	if (strstr(GetCommandLineA(), "-noplugins") != NULL)
```

Comment:
I know that this line existed prior to this PR, but using `strstr` for command line stuff is kinda bad (elaborated on a bit in #507)

I would appreciate if this could be changed? Not rly needed though, could be considered out of scope

Reply:
#507 will require one PR to remove the .txt functionality and one PR to properly incorporate the tier0 interface so imo we should let `strstr` slide for now